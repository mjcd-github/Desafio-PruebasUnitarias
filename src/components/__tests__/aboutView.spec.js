import { shallowMount } from "@vue/test-utils";
import { describe, it, expect } from "vitest";
import About from "@/views/AboutView.vue";

describe("AboutView", () => {
    it("Carga correctamente", () => {
        const wrapper = shallowMount(About);
        expect(wrapper.html()).toMatchSnapshot();
    });
});