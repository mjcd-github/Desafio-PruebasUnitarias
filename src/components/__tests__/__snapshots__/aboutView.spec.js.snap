// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AboutView > Carga correctamente 1`] = `
"<div class="about">
  <h1>This is an about page</h1>
</div>"
`;
